### ISSUE TITLE
The full screen of the video causes the height of the webview to be lost

### ISSUE THREAD

**Issue by bashen1** [iou90/react-native-autoheight-webview#178]
**Bug description:**
The page contains the video tag, and the webview adds the attribute that allowsFullscreenVideo is true. When the video is played in full screen and then closed, the height of the webview is lost

**To Reproduce:**

**Source (static HTML or url):**
```html
<video id="zkvideo" controls="controls" width="100%" height="300" class="ke-media" src="test.mp4"></video>
```
**Expected behavior:**

**Screenshots/Videos:**

**Environment:**
 - OS:Android
 - OS version:10
 - react-native version:0.61.5
 - react-native-webview version:10.4.0
 - react-native-autoheight-webview version:1.5.2

**Comment by iou90** [comment]
Should be fixed from v1.5.3.

**Comment by bashen1** [comment]
Problem solved. 🎉🎉🎉
